export-samples/html/cv-galactic-tech-innovations.html

```diff
@@ -0,0 +1,692 @@
+      }
+      .aside-section {
+        display: flex;
+        flex-direction: column;
+        font-family: Roboto, sans-serif;
+        padding: 0 var(--spacing_16) 0 var(--spacing_16);
+        gap: var(--spacing_16);
+        border-left: 1px solid var(--primary_color);
+        min-height: 100%;
+      }
+      .aside-section-name {
+        /*font-family: 'Raleway';*/
+        color: var(--primary_color);
+        font-size: 20px;
+        font-weight: 540;
+        letter-spacing: 0.96px;
+        text-transform: uppercase;
+      }
+      .aside-section-list {
+        gap: var(--spacing_16);
+        /*font-family: 'Raleway';*/
+        font-size: 15px;
+        font-weight: 500;
+        letter-spacing: 0.96px;
+        color: var(--gray_color_900);
+        list-style-type: none;
+      }
+      .aside-section-list li {
+        position: relative;
+        text-align: left;
+        line-height: 1.8em;
+      }
+      .aside-section-list li::before {
+        content: '';
+        display: block;
+        height: 7px;
+        width: 7px;
+        background: var(--primary_color);
+        border-radius: 60px;
+        position: absolute;
+        left: -20px;
+        top: 4px;
+      }
+      .aside-section-list li:first-letter {
+        text-transform: capitalize;
+      }
+      .aside-section-list a {
+        text-decoration: none;
+        letter-spacing: 0.96px;
+        color: var(--gray_color_900);
+      }
+      .aside-section-link-list-item {
+        display: flex;
+        flex-direction: row;
+        align-items: baseline;
+        gap: var(--spacing_8);
+      }
+
+      @media (min-width: 1024px) {
+        .aside-area {
+          text-align: right;
+        }
+
+        .aside-section {
+          border-left: none;
+          border-right: 1px solid var(--primary_color);
+          min-height: 100%;
+        }
+
+        .aside-section-list li {
+          text-align: right;
+        }
+        .aside-section-list li::before {
+          content: '';
+          display: block;
+          height: 0;
+          width: 0;
+        }
+        .aside-section-list li::after {
+          content: '';
+          display: block;
+          height: 7px;
+          width: 7px;
+          background: var(--primary_color);
+          border-radius: 60px;
+          position: absolute;
+          right: -20px;
+          top: 7px;
+        }
+        .aside-section-link-list-item {
+          flex-direction: row-reverse;
+        }
+      }
+
+      /* main styles */
+      .main-chapter {
+        display: flex;
+        flex-direction: column;
+        gap: var(--spacing_32);
+      }
+
+      .main-chapter-gap {
+        min-height: var(--spacing_64);
+      }
+
+      .main-chapter-header {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        min-width: fit-content;
+      }
+      .main-chapter-header-title {
+        font-size: 24px;
+        font-weight: 580;
```

Comment:
Replace by 600.
580 is not a valid weight value, the values are 100, 200, 300, 400, .... 900.

Reply:
OK